Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -389,6 +384,12 @@ end PartialOrder
 
 end IsUpper
 
+/-- If `α` is equipped with the upper topology, then it is homeomorphic to `WithUpper α`.
+-/
+def WithUpper.withUpperHomeomorph [Preorder α] [TopologicalSpace α] [IsUpper α] :
+    WithUpper α ≃ₜ α :=
+  WithUpper.ofUpper.toHomeomorphOfInducing ⟨by erw [IsUpper.topology_eq α, induced_id]; rfl⟩
```

Comment:
I think 
```suggestion
def IsUpper.withUpperHomeomorph [Preorder α] [TopologicalSpace α] [IsUpper α] :
    WithUpper α ≃ₜ α :=
  WithUpper.ofUpper.toHomeomorphOfInducing ⟨by erw [IsUpper.topology_eq α, induced_id]; rfl⟩
```
is better

Reply:
Presumably it should then be `IsUpper.withScottHomeomorph` to match?

Reply:
Done.